x-pack/test/functional/apps/uptime/locations.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { makeChecksWithStatus } from '../../../api_integration/apis/uptime/graphql/helpers/make_checks';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default ({ getPageObjects, getService }: FtrProviderContext) => {
+  const pageObjects = getPageObjects(['uptime']);
+
+  describe('location', () => {
+    const start = new Date().toISOString();
+    const end = new Date().toISOString();
+
+    const MONITOR_ID = 'location-testing-id';
+    before(async () => {
+      /**
+       * This mogrify function will strip the documents of their location
+       * data (but preserve their locaion name), which is necessary for
```

Comment:
nit s/locaion/location/

Reply:
Thanks for catching this. We will blame it on awful 2017 MBP keyboard 😆.

Reply:
Repaired in https://github.com/elastic/kibana/pull/55915/commits/eb1d96e1d89b3a8e23c72add766f5e690da6b4d8.